scripts/is/intg/helper-scripts/provision_db_is.sh

```
@@ -0,0 +1,213 @@
+fi
+
+#Run database scripts for given database engine and product version
+
+if [[ $DB_ENGINE = "postgres" ]]; then
+    # DB Engine : Postgres
+    echo "Postgres DB Engine Selected! Running WSO2-IS $WSO2_PRODUCT_VERSION DB Scripts for Postgres..."
+    export PGPASSWORD=CF_DB_PASSWORD
+    
+    # Step 1: Create databases (connect to default 'postgres' database)
+    echo "Creating databases..."
+    psql -U CF_DB_USERNAME -h CF_DB_HOST -p CF_DB_PORT -d postgres -f /opt/testgrid/workspace/$WSO2_PRODUCT_VERSION_SHORT/is_postgres_db_create.sql
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)